ngraph/test/onnx/onnx_import_exceptions.cpp

```diff
@@ -0,0 +1,95 @@
+//*****************************************************************************
+// Copyright 2017-2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//*****************************************************************************
+
+#include <exception>
+
+#include "gtest/gtest.h"
+#include "ngraph/file_util.hpp"
+#include "ngraph/frontend/onnx_import/exceptions.hpp"
+#include "ngraph/frontend/onnx_import/onnx.hpp"
+#include "ngraph/ngraph.hpp"
+#include "util/type_prop.hpp"
+
+using namespace ngraph;
+
+TEST(onnx_importer, exception_throws_ngraph_error)
```

Comment:
Any specific reason for using `TEST` instead of `NGRAPH_TEST`?

Reply:
As far as I remember the `NGRAPH_TEST` was crafted to enable the multi-backend mechanism. However I just checked it know and it turns out that also `NGRAPH_TEST` gives possibility to disable test using manifest file.

Reply:
So these tests are not using any backend nor engine, thus I considered this not necessary. Actually `type_prop` tests are written in similar fashion.